public/javascript/pump/view.js

```diff
@@ -630,6 +631,10 @@
                 streams.notifications = view.minorStreamView.model;
             }
             return streams;
+        },
+        closeDropdown: function(e) {
+            e.preventDefault();
```

Comment:
If necessary to avoid an unexpected behavior where by doing double click close and open.